metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -253,11 +253,11 @@ def _build_aggregated_conversion_node(
         logger.info(f"Recipe for conversion measure aggregation:\n{mf_pformat(conversion_measure_recipe)}")
         if base_measure_recipe is None:
             raise UnableToSatisfyQueryError(
-                f"Recipe not found for measure spec: {base_measure_spec.measure_spec} and linkable specs: {base_required_linkable_specs}"
+                f"Unable to join all items in request. Measure: {base_measure_spec.measure_spec}; Specs to join: {base_required_linkable_specs}"
             )
         if conversion_measure_recipe is None:
             raise UnableToSatisfyQueryError(
-                f"Recipe not found for measure spec: {conversion_measure_spec.measure_spec}"
+                f"Unable to build dataflow plan for measure: {conversion_measure_spec.measure_spec}"
```

Comment:
Does the measure spec indicate that this is a conversion measure? If not let's do `for conversion measure` instead of `for measure` so we can see if conversion measures are more likely to cause this problem.

Reply:
Sounds good, will update!